src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/listeners/MigrationMessageListener.kt

```diff
@@ -28,6 +29,7 @@ class MigrationMessageListener(
     val migrationMessage: MigrationMessage<*> = message.fromJson()
     when (migrationMessage.type) {
       MIGRATE_VISITS -> visitsMigrationService.migrateVisitsByPage(context(message.fromJson()))
+      MIGRATE_VISITS_BY_PAGE -> visitsMigrationService.migrateVisitsForPage(context(message.fromJson()))
```

Comment:
this reads a bit odd `MIGRATE_VISITS` calls `migrateVisitsByPage` but `MIGRATE_VISITS_BY_PAGE` calls something else

Reply:
Yes, they are sort of state transitions I suppose, 
MIGRATE_VISITS -> MIGRATE_VISITS_BY_PAGE and each function is the transition to the next stage, hence it looks odd